Tracking3D/Tracking3D.py

```diff
@@ -0,0 +1,505 @@
+                if not currentRootIDStatus:  # TODO: Remove this once this is working with the parameterNodeWrapper
+                    raise ValueError("Invalid hierarchy object selected!")
+                try:
+                    self.inProgress = True
+                    self.updateApplyButtonState()
+
+                    CT = self.ui.inputSelectorCT.currentNode()
+                    rootID = self.ui.SubjectHierarchyComboBox.currentItem()
+
+                    startFrame = self.ui.startFrame.value
+                    endFrame = self.ui.endFrame.value
+
+                    trackOnlyRoot = self.ui.onlyTrackRootNodeCheckBox.isChecked()
+
+                    self.logic.registerSequence(CT, rootID, startFrame, endFrame, trackOnlyRoot)
+                finally:
+                    self.inProgress = False
+            slicer.util.messageBox("Success!")
+        self.updateApplyButtonState()
+
+    def onImportButton(self):
+        """UI button for reading the TRA files into sequences."""
+        import glob
+        import logging
+        import os
+
+        with slicer.util.tryWithErrorDisplay("Failed to import transforms", waitCursor=True):
+            currentRootIDStatus = self.ui.SubjectHierarchyComboBox.currentItem() != 0
+            if not currentRootIDStatus:  # TODO: Remove this once this is working with the parameterNodeWrapper
+                raise ValueError("Invalid hierarchy object selected!")
+
+            CT = self.ui.inputSelectorCT.currentNode()
+            rootID = self.ui.SubjectHierarchyComboBox.currentItem()
+            rootNode = TreeNode(hierarchyID=rootID, ctSequence=CT, isRoot=True)
+
+            importDir = self.ui.ioDir.currentPath
+            if importDir == "":
+                raise ValueError("Import directory not set!")
+
+            node_list = [rootNode]
+            for node in node_list:
+                node.dataNode.SetAndObserveTransformNodeID(node.getTransform(0).GetID())
+                node_list.extend(node.childNodes)
+
+                foundFiles = glob.glob(os.path.join(importDir, f"{node.name}*.tra"))
+                if len(foundFiles) == 0:
+                    logging.warning(f"No files found matching the '{node.name}*.tra' pattern")
+                    return
+
+                if len(foundFiles) > 1:
+                    logging.warning(
+                        f"Found multiple tra files matching the '{node.name}*.tra' pattern, using {foundFiles[0]}"
+                    )
+
+                node.importTransfromsFromTRAFile(foundFiles[0])
+        slicer.util.messageBox("Success!")
+
+    def onExportButton(self):
+        """UI button for writing the sequences as TRA files."""
+        with slicer.util.tryWithErrorDisplay("Failed to export transforms.", waitCursor=True):
+            currentRootIDStatus = self.ui.SubjectHierarchyComboBox.currentItem() != 0
+            if not currentRootIDStatus:  # TODO: Remove this once this is working with the parameterNodeWrapper
```

Comment:
@NicerNewerCar  Let's discuss this and attempt to address the root cause

Reply:
I think the root issue is the type used for the `SubjectHierrarchyComboBox` in the `ParameterNodeWrapper`. The [qMRMLSubjectHierarchyComboBox](https://apidocs.slicer.org/master/classqMRMLSubjectHierarchyComboBox.html) uses the type `vtkIdType` which doesn't have an equivalent in Python as far as I can tell.

```bash
Traceback (most recent call last):
  File "C:/Users/anthony.lombardi/Projects/SAM/3d-reg-rel/lib/Slicer-5.5/qt-scripted-modules/Tracking3D.py", line 143, in setup
    self.initializeParameterNode()
  File "C:/Users/anthony.lombardi/Projects/SAM/3d-reg-rel/lib/Slicer-5.5/qt-scripted-modules/Tracking3D.py", line 187, in initializeParameterNode
    self.setParameterNode(self.logic.getParameterNode())
  File "C:/Users/anthony.lombardi/Projects/SAM/3d-reg-rel/lib/Slicer-5.5/qt-scripted-modules/Tracking3D.py", line 207, in setParameterNode
    self._parameterNodeGuiTag = self._parameterNode.connectGui(self.ui)
  File "C:\D\S5.5R\Slicer-build\bin\Python\slicer\parameterNodeWrapper\wrapper.py", line 245, in _connectGui
    _connectParametersToGui(self, paramNameToWidget)
  File "C:\D\S5.5R\Slicer-build\bin\Python\slicer\parameterNodeWrapper\wrapper.py", line 191, in _connectParametersToGui
    mappingToConnector = {
  File "C:\D\S5.5R\Slicer-build\bin\Python\slicer\parameterNodeWrapper\wrapper.py", line 192, in <dictcomp>
    paramName: value if isinstance(value, GuiConnector) else createGuiConnector(value, self.dataType(paramName))
  File "C:\D\S5.5R\Slicer-build\bin\Python\slicer\parameterNodeWrapper\guiConnectors.py", line 154, in createGuiConnector
    raise RuntimeError(f"Unable to create GUI connector from datatype '{datatype}' to widget type '{type(widget)}'")
RuntimeError: Unable to create GUI connector from datatype '<class 'str'>' to widget type '<class'PythonQt.qSlicerSubjectHierarchyModuleWidgets.qMRMLSubjectHierarchyComboBox'>'
```

Reply:
From [vtk user email list](https://public.kitware.com/pipermail/vtkusers/2010-January/056399.html)
> But as for for vtkIdType, it's just a typedef as Jerome says.  In Python, you can use a python "int" or "long" for vtkIdType.

Unfortunately this results in the same error as above when the type is set to `int`.

Reply:
There does exist a `vtk.VTK_ID_TYPE` constant however this fails the typecheck:
```python
my_var: vtk.VTK_ID_TYPE
```
Results in 
```bash
TypeError: Annotated[t, ...]: t must be a type. Got 12.
```

Confirmed:
``` python
>>> import vtk
>>> vtk.VTK_ID_TYPE
12
>>> 
```

Reply:
@jcfr To reproduce:

Uncomment the following lines then enter the Tracking3D module
https://github.com/BrownBiomechanics/SlicerAutoscoperM/blob/f377ef49140e6610d43598384edc74ff22695453/Tracking3D/Resources/UI/Tracking3D.ui#L81-L83